### PR TITLE
Dm 2069

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2069

## Description - what does this code do?
Adds new section links and buttons to practice view side nav.

## Testing done - how did you test it/steps on how can another person can test it 
1. Log in as a NON-admin and visit a practice's show page that you did not create.
2. Make sure you don't see the `Edit practice` button in the side nav.
3. Log out and log back in as an admin.
4. Make sure the `Edit practice` button is now displayed.
5. Make sure the `Email practice` button sends an email to the practice's `support_network_email` and that any 
    `practice_emails` are added in the `cc` section.
6. Scroll down to the `Overview` section and make sure the `Overview` side nav link is highlighted.
7. Click on each side nav section link and make sure you are navigated to that section.
8. Scroll down the page and make sure the sections in the side nav are highlighted as you reach them.
9. Make sure the practice's `tagline` is now gone along with the legacy `Edit` button.
10. Inspect the practice's show view with smaller viewports and make sure the `Email`, `Adopt`, and `Edit`(if admin/practice owner) are fixed to the bottom of the page.
11. Confirm styling matches Figma designs.

## Screenshots, Gifs, Videos from application (if applicable)
Screen Shot 2020-09-20 at 8 02 57 PM

Screen Shot 2020-09-20 at 8 03 21 PM

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/FogT5mBUVl8rE7OgWDQWeJ/VADM?node-id=6945%3A34

## Acceptance criteria
- Build out side nav with sections
- Overview
- Implementation
- Contact
- About

- Build Buttons below nav
- Email Practice (white)
- Contact opens up email with main email address and cc (Mailto: link)
- Start Adoption (Blue)
- “Edit Your Practice” (if they’re the practice owner) 
- When you click on a section you’re taken to that section
- Nav scrolls as you scroll
- Nav highlights the section you’re on as your scroll through different sections
- On mobile view this doesn’t display, and just have fixed footer with buttons 
- Email
- Adopt
- Edit

- Remove tagline (It’s floating with edit button)

## Definition of done
- [X] Unit tests written (if applicable)
- [X] e2e/accessibility tests written (if applicable)
- [X] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/views/practices/show/authenticated/_show_authenticated.html.erb [hunk 1]
```diff
@@ -48,17 +24,36 @@
...
+          <i class="fas fa-edit margin-right-1"></i>Edit practice
+        <% end %>
+      </div>
+    <% end %>
   </div>
 </div>
+
 <div class="practice-mobile-nav padding-1 z-top desktop:display-none display-flex justify-center">
-  <%= link_to 'See adoption checklist', practice_planning_checklist_url(practice_id: @practice.slug), id: 'next-steps-link-in-mobile-nav', class: 'usa-button' %>
+  <%= mail_to main_email, 'Email', cc: practice_emails.map(&:address).join(', '), id: 'email-link-in-mobile-nav', class: 'width-100 usa-button--outline dm-btn-primary display-flex flex-align-center flex-justify-center height-5 x0-875-right', 'aria-label': "Mobile nav email #{main_email}"  %>
```
Comment (ahay-agile6) [inline app/views/practices/show/authenticated/_show_authenticated.html.erb:48]: how many times is this `practice_emails.map(&:address).join(', ')` used on the page?
can we make this a variable to use in those spots?

**Review by ahay-agile6 — CHANGES_REQUESTED** [review]
lgtm functionality wise. just one code quality question

File: app/views/practices/show/authenticated/_show_authenticated.html.erb (hunk 1 above)
Comment (JoshingYou1) [reply app/views/practices/show/authenticated/_show_authenticated.html.erb]: I created a variable for this one as well as a couple of others. Thank you for catching that!

**Review by ahay-agile6 — APPROVED** [review]

**Review by emyl3 — APPROVED** [review]
lol not sure why there is a random `rails` file with no changes in the diff 🤷 🤷‍♂️ 
this looks good to me and worked as intended! thank you josh!

**Pull request merged**